css/styles.css

```diff
@@ -103,11 +103,13 @@ body {
   opacity: .5;
 }
 
-#logo {
+.logo img {
   width: 30px;
   height: 30px;
 
-  fill: #4957BB;
+  /* fill: #4957BB; */
+  /* https://codepen.io/sosuke/pen/Pjoqqp */
+  filter: invert(32%) sepia(56%) saturate(1172%) hue-rotate(203deg) brightness(91%) contrast(88%);
```

Comment:
Was this necessary for some reason because the SVGs are now in files?

Reply:
This is a replacement for the fill attribute, since you can't technically set the fill color of the SVG without using javascript.